test/parser.test.js

```diff
@@ -1,7 +1,10 @@
 const test = require('ava')
+const nock = require('nock')
```

Comment:
I think this is leftover as `nock` isn't used anywhere in this file.

Reply:
fixed